pkg/operator/ceph/object/rgw-probe.sh

```diff
@@ -17,11 +17,12 @@ RGW_URL="$PROBE_PROTOCOL://0.0.0.0:$PROBE_PORT"
 
 function check() {
   local URL="$1"
+  # --max-time 3 - override the default 2 minutes wait time down to 3 seconds
```

Comment:
I'd like to make sure we capture the reason why we are using this flag in this case, instead of just stating what the flag does.

```suggestion
  # --max-time - curl doesn't always respond to SIGTERM on timeout and can become zombie; make sure it times out
```